pkg/archive/untar.go

```diff
@@ -61,7 +63,6 @@ func untar(r io.Reader, dir string) (err error) {
 		}
 		baseFile := filepath.Base(f.Name)
 		abs := path.Join(dir, baseFile)
-		fmt.Println(abs, f.Name)
```

Comment:
I'm removing this because it gets in the way of the progress bar (if you print something to stdout before closing the progress bar, it messes up the output). More importantly, it doesn't feel like getting the list of tmp folders extracted is that useful for the end user.

Reply:
Isn't the progress bar there for the download of a file? Then when the download is complete, it should stop /complete.

Why is the unzip / untar operation part of the progress bar's scope, if it's only there for the download?

Is it a timing issue? That the progress bar prints some additional output after the download has "finished"?

Reply:
There's no "download of a file" for the unzip/untar function in the existing implementation.  `archive.untar` receives the response body (wrapped as a `NopCloser`), and handles all the download internally (https://github.com/alexellis/arkade/blob/master/pkg/archive/untar.go#L44). That's why I went with this approach, where the reader is wrapped in the progress bar. 

If we want to scope the progress bar only to the download step, we could change `get.Download` to first download the file for all cases, and then do the format specific manipulations (e.g. if it's a tar.gz, untar and unzip; for a binary, just move to the final destination).  What approach do you prefer?